src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3736,6 +3736,18 @@ impl AstDisplay for StartTransactionStatement {
 }
 impl_display!(StartTransactionStatement);
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct ShowCreateTypeStatement<T: AstInfo> {
+    pub type_name: T::DataType,
+}
+
+impl<T: AstInfo> AstDisplay for ShowCreateTypeStatement<T> {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        f.write_str("SHOW CREATE TYPE");
+        f.write_node(&self.type_name);
```

Comment:
Space missing, as shown by the `test_parser` fail in CI, where this printed `SHOW CREATE TYPEfoo`.